packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js

```diff
@@ -68,12 +69,7 @@ export default class AgoraIpV4OrderController {
     this.user = this.$state.params.user;
     this.catalogName = this.$state.params.catalogName;
     this.loadServices();
-  }
-
-  getIpv4Catalog() {
-    return this.ipCatalog.filter(
-      (plan) => plan.planCode.match(/^ip-v4.*|ip-failover.*/) != null,
-    );
+    this.ipv4Catalog = this.getIpv4Catalog();
```

Comment:
Do you need this line ? I don't see ipv4Catalog used

Reply:
Its needed for second commit.